R/auditCommonFunctions.R

```diff
@@ -1471,9 +1471,9 @@
     .jfaTableNumberUpdate(jaspResults)
     tb <- createJaspTable(gettextf("<b>Table %1$i.</b> Default Settings Audit Risk Model", jaspResults[["tabNumber"]]$object))
     tb$addColumnInfo(name = "category", title = "", type = "string")
-    tb$addColumnInfo(name = "ir", title = gettext("Inherent risk"), type = "number", format = "pc")
-    tb$addColumnInfo(name = "cr", title = gettext("Control risk"), type = "number", format = "pc")
-    tb$addColumnInfo(name = "car", title = gettext("Analytical risk"), type = "number", format = "pc")
+    tb$addColumnInfo(name = "ir", title = gettext("Inherent risk"), type = "number", format = "pc;dp:0")
+    tb$addColumnInfo(name = "cr", title = gettext("Control risk"), type = "number", format = "pc;dp:0")
+    tb$addColumnInfo(name = "car", title = gettext("Analytical risk"), type = "number", format = "pc;dp:0")
```

Comment:
```suggestion
    tb$addColumnInfo(name = "ir", title = gettext("Inherent risk"), type = "number", format = "pc:0")
    tb$addColumnInfo(name = "cr", title = gettext("Control risk"), type = "number", format = "pc:0")
    tb$addColumnInfo(name = "car", title = gettext("Analytical risk"), type = "number", format = "pc:0")
```

Is also possible. However if the thing you did works that is also fine ofc

Reply:
OK, implemented!